docs/feature-flagging/concepts/mutual_exclusion.md

```diff
@@ -57,6 +57,64 @@ Here you are able to define the Experiment name, the Variations to include in th
 Once your Experiment allocation is created, you'll want to add an Experiment Analysis to it to understand how it's performing. Do this by clicking on the `Create Analysis` button and filling out information about the Analysis. See [Creating Experiments](/experiment-analysis/configuration) for more information.
 ![Creating an experiment analysis](/img/feature-flagging/layer-experiment-analysis.png)
 
+## Setting up the Eppo SDK for a Layer
+
+A Layer in Eppo will use the `getJSONAssignment()` SDK method along with the Layer key as the Flag key, and will return the parameters associated with the assignment for that user as an array of objects.
+
+For example:
+
+```javascript
+eppoClient.getAssignment(
+    subjectKey,
+    layerKey,
+    subjectAttributes,
+    defaultValue
+)
+```
+
+will return:
+
+```javascript
+[
+  { parameterName: 'sticky_banner', variationValue: 'false' },
+  { parameterName: 'banner_cta', variationValue: 'Sign in for updates on shipping' },
+  { parameterName: 'shoprunner', variationValue: 'true' },
+  { parameterName: 'promo', variationValue: '$20 off $100' }
```

Comment:
@aarsilv and I want to change this return type because it is extremely unwieldy for users, to instead return

```
{'sticky_banner': 'false', 'banner_cta': 'Sign in for updates on shipping', ...}
```

Reply:
sweet! good to know and I agree. I'll keep this as is for now so we can highlight to users the difference between a JSON flag and a layer.

Reply:
we basically do not want anyone adopting this approach as it would make it impossible for us to make this change